demo/trigger.py

```diff
@@ -0,0 +1,31 @@
+import requests
+import json
+
+# The URL for the PapermillHub API endpoint
+papermill_url = "http://localhost:8000/services/papermillhub/api/jobs/"
+# A token that the PapermillHub service can use to authenticat
+# with JupyterHub
+papermillhub_token = "super-secret"
+
+# A test username
+user = "user"
+
+# Create a user token, which can supsequently be used to invoke
+# papermillhub as if the user were launching the job.
+note = "Created by PapermillHub"
```

Comment:
At the moment, the workflow here is to request a *user* token, and then use that to trigger the papermillhub job. We may want to rethink that by specifying the user in the url, then having papermillhub trigger the user job internally. I haven't really thought through the pros/cons of that yet.